app.cfg.example

```diff
@@ -161,7 +161,11 @@ no_build_permission_comment = Label `bot:build` has been set by user `{build_lab
 allow_update_submit_opts = false
 
 # defines which name-value pairs (environment variables) are allowed to be
-# exported into the build environment via `exportvariable` filters
+# exported into the build environment via 'exportvariable' filters
+# The bot build script makes use of the variable 'SKIP_TESTS' to determine if
+# ReFrame tests shall be skipped or not. Default value is 'no'. If the value is
+# 'yes' and the exportvariable filter is added to a bot build command
+# ('export:SKIP_TESTS=yes'), ReFrame tests are skipped.
 allowed_exportvars = ["NAME1=value_1a", "NAME1=value_1b", "NAME2=value_2"]
```

Comment:
```suggestion
#allowed_exportvars = ["NAME1=value_1a", "NAME1=value_1b", "NAME2=value_2"]
```
This is not a change in this PR and a minor thing, but shouldn't `allowed_exportvars` be commented out by default in the example file (i.e., by default the dummy variables in the example aren't exported)

Reply:
While it's only an example file, it's a good idea to comment this and add a note ala "This setting is optional. If you want to use it uncomment it and provide meaningful key-value pair(s)."